<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManagerSystem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font-family:&quot;Roboto&quot;,&quot;Segoe UI&quot;,sans-serif;
	font-size:12px;
	color:#ffffff;
	background-color:#424242;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QToolButton" name="btnStudentInfo">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>学生信息</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/【哲风壁纸】AI-少女.png</normaloff>:/【哲风壁纸】AI-少女.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSchedule">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>课程安排</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/【哲风壁纸】动漫美女-桃花.png</normaloff>:/【哲风壁纸】动漫美女-桃花.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnFinance">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>财务</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/aaa.jpg</normaloff>:/aaa.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnHonor">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>荣誉墙</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/hhh.jpg</normaloff>:/hhh.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSettings">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>系统设置</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/【哲风壁纸】Arch-linux系统.png</normaloff>:/【哲风壁纸】Arch-linux系统.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="StudentInfoWidget" name="pageStudentInfo"/>
      <widget class="QWidget" name="pageSchedule">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>230</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageFinence">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>190</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageHonor">
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>200</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSettings">
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>250</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StudentInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">studentinfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
